stacks/dhis2/helmfile.yaml

```diff
@@ -53,10 +55,11 @@ releases:
     version: 11.0.4
     verify: false
     values:
-      - primary:
-          podLabels:
-            im-data-id: "{{ requiredEnv "INSTANCE_ID" }}"
+      - commonLabels:
+          im-id: "{{ requiredEnv "INSTANCE_ID" }}"
+          im-type: "db"
```

Comment:
On one hand, I want to choose something less generic such as postgresql. But on the other, I want to cater to less technical users which just knows it's a database. What do you think?

We might also think about how the user even know which types they can query by.

Reply:
I chose db as it's consice. We can go either way